x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/package_policy_input_config.tsx

```diff
@@ -105,7 +105,8 @@ export const PackagePolicyInputConfig: React.FunctionComponent<{
           <EuiFlexGroup direction="column" gutterSize="m">
             {requiredVars.map((varDef) => {
               const { name: varName, type: varType } = varDef;
-              const { value, frozen } = packagePolicyInput.vars![varName];
+              if (!packagePolicyInput.vars) return;
+              const { value, frozen } = packagePolicyInput.vars[varName];
```

Comment:
Added these nullchecks as I kept getting js undefined errors while testing locally the upgrade page of aws. There is a short time initially when those variables are undefined, making the form unusable.

<img width="964" alt="image" src="https://user-images.githubusercontent.com/90178898/153204708-5fe7773b-02f1-45ef-9bb2-c725442f4fba.png">